routeoptimization/snippets/src/main/java/com/example/OptimizeTours.java

```diff
@@ -22,14 +22,17 @@
 package com.example;
 
 // [START routeoptimization_v1_OptimizeTours_sync]
+
 import com.google.maps.routeoptimization.v1.OptimizeToursRequest;
 import com.google.maps.routeoptimization.v1.OptimizeToursResponse;
 import com.google.maps.routeoptimization.v1.RouteOptimizationClient;
+import com.google.maps.routeoptimization.v1.RouteOptimizationSettings;
 import com.google.maps.routeoptimization.v1.Shipment;
 import com.google.maps.routeoptimization.v1.Shipment.VisitRequest;
 import com.google.maps.routeoptimization.v1.ShipmentModel;
 import com.google.maps.routeoptimization.v1.Vehicle;
 import com.google.type.LatLng;
+import org.threeten.bp.Duration;
```

Comment:
@bshi does this sample need to pull this in? Can we use java.time.Duration instead?